src/index.html

```diff
@@ -11,8 +11,53 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="imac image"
+      />
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
```

Comment:
- Use semantic tags where possible
<img width="967" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/eeba4f58-75de-4951-8442-b8a6f526ab8f">

- alt attribute should describe the image content
<img width="468" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/35346262-2849-4a20-8618-7dae1127d4dc">